data_validation/cli_tools.py

```diff
@@ -581,18 +589,21 @@ def _configure_custom_query_parser(custom_query_parser):
     )
 
 
-def _add_common_arguments(parser):
+def _add_common_arguments(parser, query_type="ANY"):
```

Comment:
We don't need the query_type parameter any more.

Reply:
Fixed [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/607/commits/7667b5f603a4796bb96360ec48b7c66a9611179e)